contrib/cross-compile-armv6hf/docker-build.sh

```diff
@@ -1,14 +1,17 @@
 #!/usr/bin/env bash
 set -eux
 
-PI1_TOOLS_DIR="/pi/tools/arm-bcm2708/arm-linux-gnueabihf"
+cargo install --force --locked bindgen-cli
+
+PI1_TOOLS_DIR=/pi/tools/arm-bcm2708/arm-linux-gnueabihf
+PI1_TOOLS_SYSROOT_DIR=$PI1_TOOLS_DIR/arm-linux-gnueabihf/sysroot
```

Comment:
Just out of curiosity. This path extend to `/pi/tools/arm-bcm2708/arm-linux-gnueabihf/arm-linux-gnueabihf/sysroot`. Is it intentional that `arm-linux-gnueabihf` is repeated in the path? It seemed to be previously the same, but it feels a bit off, so I just wanted to ask if that is the expected behavior :)

Reply:
Yes - this is how the files are arranged in the https://github.com/raspberrypi/tools repository.

Please note that `/pi/tools/arm-bcm2708/arm-linux-gnueabihf` is a symlink to `/pi/tools/arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf`, so contents of the sysroot directory comes from: https://github.com/raspberrypi/tools/tree/master/arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf/arm-linux-gnueabihf/sysroot

I must admit that I am not so fluent in cross-compilation to fully understand what is where in this structure. :disappointed:

Reply:
Ah oki, interesting. Just looks a bit like a minor mistake from the outside, but all good then^^